assets/css/style.css

```diff
@@ -40,6 +40,12 @@ p {
 	font-weight: 800;
 }
 
+.brewery {
```

Comment:
brewery is so rad but breweries are better

Reply:
disagree.... sorry :P